lib/ui/login/login_screen.dart

```diff
@@ -182,6 +189,23 @@ class _LoginScreenState extends State<LoginScreen>
     );
   }
 
+  // TODO: remove this dummy method in [#10]
+  void login() async {
```

Comment:
Optinal: action should start with a verb too

```suggestion
  void logIn() async {
```